pb/storage_service.pb.go

```diff
@@ -550,6 +550,187 @@ func (x *StorageGetResponse) GetStorage() []byte {
 	return nil
 }
 
+type GetPowerflexVolumesRequest struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	VolumeName []string `protobuf:"bytes,1,rep,name=volumeName,proto3" json:"volumeName,omitempty"`
+	SystemId   string   `protobuf:"bytes,2,opt,name=systemId,proto3" json:"systemId,omitempty"`
+}
+
+func (x *GetPowerflexVolumesRequest) Reset() {
+	*x = GetPowerflexVolumesRequest{}
+	if protoimpl.UnsafeEnabled {
+		mi := &file_pb_storage_service_proto_msgTypes[10]
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		ms.StoreMessageInfo(mi)
+	}
+}
+
+func (x *GetPowerflexVolumesRequest) String() string {
+	return protoimpl.X.MessageStringOf(x)
+}
+
+func (*GetPowerflexVolumesRequest) ProtoMessage() {}
+
+func (x *GetPowerflexVolumesRequest) ProtoReflect() protoreflect.Message {
+	mi := &file_pb_storage_service_proto_msgTypes[10]
+	if protoimpl.UnsafeEnabled && x != nil {
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		if ms.LoadMessageInfo() == nil {
+			ms.StoreMessageInfo(mi)
+		}
+		return ms
+	}
+	return mi.MessageOf(x)
+}
+
+// Deprecated: Use GetPowerflexVolumesRequest.ProtoReflect.Descriptor instead.
```

Comment:
I see a few deprecated comments in this file. Is that intentional or could these functions be renamed?

Reply:
I am not sure about this. This type of comment is in all of the other generated code for other services.